addons/website/static/src/snippets/s_image_gallery/002.scss

```diff
@@ -0,0 +1,271 @@
+.s_image_gallery[data-vcss="002"] {
+    &.o_grid, &.o_masonry {
+        .img {
+            width: 100%;
+        }
+    }
+    &.o_grid {
+        &.o_spc-none div.row {
+            margin-bottom: 0px;
+        }
+        &.o_spc-small div.row > div {
+            margin-bottom: $spacer;
+        }
+        &.o_spc-medium div.row > div {
+            margin-bottom: $spacer * 2;
+        }
+        &.o_spc-big div.row > div {
+            margin-bottom: $spacer * 3;
+        }
+    }
+    &.o_masonry {
+        &.o_spc-none div.o_masonry_col {
+            padding: 0;
+            > img, > a > img, > .media_iframe_video {
+                margin: 0 !important;
+            }
+        }
+        &.o_spc-small div.o_masonry_col {
+            padding: 0 ($spacer * .5);
+            > img, > a > img, > .media_iframe_video {
+                margin-bottom: $spacer !important;
+            }
+        }
+        &.o_spc-medium div.o_masonry_col {
+            padding: 0 $spacer;
+            > img, > a > img, > .media_iframe_video {
+                margin-bottom: $spacer * 2 !important;
+            }
+        }
+        &.o_spc-big div.o_masonry_col {
+            padding: 0 ($spacer * 1.5);
+            > img, > a > img, > .media_iframe_video {
+                margin-bottom: $spacer * 3 !important;
+            }
+        }
+    }
+    &.o_nomode {
+        &.o_spc-none .row > div {
+            padding-top: 0;
+            padding-bottom: 0;
+        }
+        &.o_spc-small .row > div {
+            padding-top: $spacer * .5;
+            padding-bottom: $spacer * .5;
+        }
+        &.o_spc-medium .row > div {
+            padding-top: $spacer;
+            padding-bottom: $spacer;
+        }
```

Comment:
Yes it's because it's the initial scss file that I duplicated first in this commit and then I updated it in the second commit (cf. https://github.com/odoo/odoo/pull/172727#:~:text=Why%20two%20commits,this%20new%20versioning. 😉 )
So reviewing this file **in this commit** is not necessary 🙂

Reply:
Yes, I know ;) But if you move code around you have to adapt that accordingly.
Otherwise you move code in a specific `[MOV]` commit... git it's usually smart enough to understand what you're doing in this case  😉

Reply:
The goal is to squash https://github.com/odoo/odoo/pull/172727/commits/78ad286b8d073579469c656e78d9d3bb36236a55 and https://github.com/odoo/odoo/pull/172727/commits/9b302fdde7e205e6403cc93b5e4053c000b9aeb4 together once the design review will be approved, so this kind of occurrences should disappear after 🙂  
I've split it into two commits to help you see exactly what's been changed 🙂 .